examples/python_scripts/stacked_line_graphs.py

```diff
@@ -0,0 +1,90 @@
+import sys
+
+sys.path.append("/usr/gapps/spot/dev/hatchet-venv/x86_64/lib/python3.9/site-packages/")
+sys.path.append("/usr/gapps/spot/dev/thicket-playground-dev/")
+sys.path.append("/usr/gapps/spot/dev/hatchet/x86_64/")
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import hatchet as ht
+import thicket as th
+import copy
+import re
+import argparse
+
+from IPython.display import display, HTML
+from glob import glob
+from hatchet import QueryMatcher
```

Comment:
I think the `from`s can technically go at the end of each block, but this is pep8
```suggestion
import argparse
import copy
from glob import glob
import re
import sys

sys.path.append("/usr/gapps/spot/dev/hatchet-venv/x86_64/lib/python3.9/site-packages/")
sys.path.append("/usr/gapps/spot/dev/hatchet/x86_64/")
sys.path.append("/usr/gapps/spot/dev/thicket-playground-dev/")

import hatchet as ht
from hatchet import QueryMatcher
from IPython.display import display, HTML
import matplotlib.pyplot as plt
import pandas as pd

import thicket as th
```

Reply:
The reason why Hatchet and Thicket are separate is because we consider Hatchet a third-party package in relation to Thicket